test/unittests/mongoBackend/mongoOntimeintervalOperations_test.cpp

```diff
@@ -522,8 +526,10 @@ TEST(mongoOntimeintervalOperations, mongoGetContextElementResponses_dbfail)
               "attrNames: { $in: [ \"A1\", \"A2\", \"A3\", \"A4\" ] } }, orderby: { creDate: 1 } } - "
               "exception: boom!!)", err);
 
-    /* Release mock */
-    setMongoConnectionForUnitTest(NULL);
+    /* Restore real DB connection */
+    setMongoConnectionForUnitTest(connectionDb);
```

Comment:
They are used to "recover" the actual connection to DB (previously overriden by `setMongoConnectionForUnitTest(connectionMock)`) so next next find it correctly (otherwise, the next test will bet `connectionMock` and that crashed most of the times).

I think is not related with utInit/utExit (these functions create and release mocks, but they don't do anything related connection objects, as far as I remember...).